src/vs/workbench/browser/layout.ts

```diff
@@ -826,12 +892,14 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 		}));
 	}
 
+	getClientArea(): Dimension {
+		const dim = getClientArea(this.parent);
+		return this.state.windowBorder ? new Dimension(dim.width - 2, dim.height - 2) : dim;
+	}
+
 	layout(): void {
 		if (!this.disposed) {
-			this._dimension = getClientArea(this.parent);
-
-			position(this.container, 0, 0, 0, 0, 'relative');
```

Comment:
@eamodio why is this removed?

Reply:
By using `vh/wh` in the CSS we don't need to manually size it any more